### DISCUSSION TITLE
Do we have plan release the Poseidon v17?

### DISCUSSION THREAD

**Discussion by nesgarbo** [primefaces/community#771] [PrimeNG Templates] [answered]
Hi, I noticed that many templates have been updated to v17. Will Poseidon be updated too?

**Comment by cetincakiroglu** [comment]
Hi,

Poseidon will not receive a v17 update because it's currently a legacy template of ours. However, we're preparing a guide for "update template manually". We'll probably add the guide to the primeng.org in next week

**Comment by cetincakiroglu** [comment, marked as answer]
Hi @nesgarbo,

There has been a mix-up in our legacy template lists, as we forgot to remove Poseidon from the list. Poseidon is currently being updated, and we plan to release it next week.

**Reply by vigalo** [reply]
Hi, @cetincakiroglu :

Will be released as expected the Poseidon template update this week?

Thanks.

**Reply by cetincakiroglu** [reply]
Hi,

We're sorry for the delay, Poseidon is in currently review. We're planning to release upcoming weeks.

**Comment by cetincakiroglu** [comment]
Hi,

You can download poseidon 17 from the store.

**Reply by AshishFreak** [reply]
Hi @cetincakiroglu  i am student right now can you please share poseidon thee with it will help me alot ?

**Reply by AshishFreak** [reply]
where is store ? can you share link

**Reply by AshishFreak** [reply]
Sir i cant afford from store

**Reply by AshishFreak** [reply]
Sir i request you please i know i can use sakai but please i need posoeidon or apollo it will be really a help
